compiler/luci/service/include/luci/Service/CircleTypeInference.h

```diff
@@ -63,6 +63,7 @@ class Algorithm final : public luci::CircleNodeVisitor<loco::DataType>
   // loco::DataType visit(const luci::CircleAbs *node) final;
   // loco::DataType visit(const luci::CircleAdd *node) final;
   // loco::DataType visit(const luci::CircleAddN *node) final;
+  // loco::DataType visit(const luci::CircleAll *node) final;
```

Comment:
Hum? where the `CircleAll` came from?

Reply:
I definitely removed it but why.... @.@
I will remove it :(